<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Address</class>
 <widget class="QMainWindow" name="Address">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>小型学生通讯录管理系统V1.0 @Ryzin.</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(170, 219, 255);</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>小型学生通讯录管理系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>160</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="14,1,1">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>Last updated 20180415</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="15,2,1">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_loginMode">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>登陆状态：未登陆</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="action_V1_0_A"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>选项(&amp;O)</string>
    </property>
    <addaction name="action_L_2"/>
    <addaction name="action_R"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>账户(&amp;A)</string>
    </property>
    <addaction name="action_L"/>
    <addaction name="action_O_4"/>
    <addaction name="action_Q"/>
   </widget>
   <addaction name="menu_A"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="text">
    <string>创建通讯录(&amp;N)</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>查看通讯录文件(&amp;V)</string>
   </property>
  </action>
  <action name="action_O_2">
   <property name="text">
    <string>打开通讯录文件(&amp;W)</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>最近访问的通讯录(&amp;F)</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>如何使用(&amp;H)</string>
   </property>
  </action>
  <action name="action_V1_0_A">
   <property name="text">
    <string>关于小型学生通讯录管理系统V1.0(&amp;A)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出(Q)</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>添加(&amp;A)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>查看与修改(&amp;F)</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>修改(&amp;R)</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>初始化通讯录(&amp;R)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>登陆账户(&amp;I)</string>
   </property>
  </action>
  <action name="action_O_3">
   <property name="text">
    <string>注销账户(&amp;O</string>
   </property>
  </action>
  <action name="action_O_4">
   <property name="text">
    <string>注销账户(&amp;O)</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>升序</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>降序</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>升序</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>降序</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>升序</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>降序</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>升序</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>降序</string>
   </property>
  </action>
  <action name="action_Changepwd">
   <property name="text">
    <string>修改密码(&amp;R)</string>
   </property>
  </action>
  <action name="action_L_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>语言(&amp;L)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
